rulesets/SlevomatCodingStandard/Sniffs/Types/EmptyLinesAroundTypeBracesSniff.php

```diff
@@ -0,0 +1,163 @@
+<?php declare(strict_types = 1);
```

Comment:
`declare(strict_types = 1);` is not a part of our rule set why you add them in all files?

Reply:
as it states in REAMDE - those file are just copy from other repository.
i don't want to use that repository as dependency, because it has composer with other dependencies, which we don't neet